dependencies.yaml

```diff
@@ -591,20 +595,37 @@ dependencies:
               - pynvjitlink>=0.0.0a0
```

Comment:
Anchor this.
```suggestion
              - &pynvjitlink_unsuffixed pynvjitlink>=0.0.0a0
```

Reply:
ah right, agreed. Updated in recent commits (I did this locally instead of in the browser to also be able to re-run `rapids-dependency-file-generator`, just to check).